Mathlib/Order/GameAdd.lean

```diff
@@ -19,36 +19,68 @@ This file defines, given relations `rα : α → α → Prop` and `rβ : β →
 decreasing either entry (with respect to `rα` and `rβ`). It is so called since it models the
 subsequency relation on the addition of combinatorial games.
 
+We also define `Sym2.GameAdd`, which is the unordered pair analog of `Prod.GameAdd`.
+
 ## Main definitions and results
 
 - `Prod.GameAdd`: the game addition relation on ordered pairs.
 - `WellFounded.prod_gameAdd`: formalizes induction on ordered pairs, where exactly one entry
   decreases at a time.
 
-## Todo
-
-- Add custom `induction` and `fix` lemmas.
-- Define `Sym2.GameAdd`.
+- `Sym2.GameAdd`: the game addition relation on unordered pairs.
+- `WellFounded.sym2_gameAdd`: formalizes induction on unordered pairs, where exactly one entry
+decreases at a time.
 -/
 
 
-variable (rα : α → α → Prop) (rβ : β → β → Prop)
+variable {α β : Type _} {rα : α → α → Prop} {rβ : β → β → Prop}
 
```

Comment:
Lean3 has `/-! ### `prod.game_add` -/` here between the `variable` and `namespace` line